tests/Http/SavedSearches/UpdateSavedSearchRequestHandlerTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\SavedSearches;
+
+use Broadway\CommandHandling\Testing\TraceableCommandBus;
+use CultuurNet\UDB3\Deserializer\MissingValueException;
+use CultuurNet\UDB3\Http\ApiProblem\AssertApiProblemTrait;
+use CultuurNet\UDB3\Http\Request\Psr7RequestBuilder;
+use CultuurNet\UDB3\Http\Response\AssertJsonResponseTrait;
+use CultuurNet\UDB3\SavedSearches\Command\UpdateSavedSearch;
+use CultuurNet\UDB3\SavedSearches\Properties\QueryString;
+use Fig\Http\Message\StatusCodeInterface;
+use PHPUnit\Framework\TestCase;
+
+class UpdateSavedSearchRequestHandlerTest extends TestCase
+{
+    use AssertApiProblemTrait;
+    use AssertJsonResponseTrait;
+
+    private const USER_ID = 'b9dc94df-c96b-4b71-8880-bd46e4e9a644';
+
+    private TraceableCommandBus $commandBus;
+
+    private UpdateSavedSearchRequestHandler $updateSavedSearchRequestHandler;
+
+    private Psr7RequestBuilder $psr7RequestBuilder;
+
+    protected function setUp(): void
+    {
+        $this->commandBus = new TraceableCommandBus();
+
+        $this->updateSavedSearchRequestHandler = new UpdateSavedSearchRequestHandler(
+            self::USER_ID,
+            $this->commandBus
+        );
+
+        $this->psr7RequestBuilder = new Psr7RequestBuilder();
+
+        $this->commandBus->record();
+    }
+
+
+    /**
+     * @test
+     */
+    public function it_can_save_a_search(): void
```

Comment:
This test is kinda strange to me, you are doing a create to the `POST` endpoint, while it should use the `PUT` endpoint.
Also, the happy path of a successful change is not tested.